### ISSUE TITLE
Charts and Graphs not working

### ISSUE THREAD

**Issue by count00zero** [ampache/ampache#3379]
### Describe the bug
When enabling statistical graphs (statistical_graphs = "true") and installing pchart (sudo -u www-data composer2 require szymach/c-pchart "2.*") charts should be displayed, but are not (see screenshot)

### To reproduce

1. Login
2. Click on Home -> 'Statistics' (under information)
3. Click on missing Statistic graph ("broken" image)
4. See blank page

### Expected behavior
See statistical graphs

### Screenshots
![Clipboard02](https://user-images.githubusercontent.com/14225157/199974635-e3643458-8d8a-4d5b-bcfb-3269af44cf90.jpg)

## Environment

Installed Ampache version: 1fb12595fc9cb27dd96f0616936a1e7d5f51e81e.
GitHub Branch: "develop"

PHP: 8.1.12

nginx version: nginx/1.18.0

Server operating system: ubuntu mate 22.04 
Client operating system: ubuntu mate 22.04 & windows 11

### Client type

* Web
* Ampache

## Settings
`statistical_graphs = "true"`

## Logs

```
2022-11-04T13:38:18+01:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\StatisticGraph\ShowAction" for action "show"
2022-11-04T13:38:18+01:00 [martin] (Ampache\Module\Application\StatisticGraph\ShowAction) -> Access denied, statistical graph disabled.

```

**Comment by lachlan-00** [comment]
6d18d5975 fixes that.
the config should also say `composer require szymach/c-pchart "3.*"`
or `composer install --dev`

now too so i'll update that

**Comment by count00zero** [comment]
Great & thank you very much!

**Comment by count00zero** [comment]
After some testing I discovered a funny situation.

If I enable statistics `statistical_graphs = "true"`, the statistics-graph is just empty (no chart, no google-maps, etc.):
![grafik](https://user-images.githubusercontent.com/14225157/202656767-9173b690-53ff-4355-ae03-14d8140f39c4.png)

Maybe that log line is connected to that:
`2022-11-18T09:28:28+01:00 [ampache] (log.lib) -> [Runtime Error] Attempt to read property "id" on string in file /opt/ampache/src/Module/Statistics/Stats.php(941)`

If I disable statistics, maps is back and the "broken" chart-icons (clicking on those opens empty page - as it should):
![grafik](https://user-images.githubusercontent.com/14225157/202657124-7f91715f-e28b-42ae-9c56-c893d0f4e94f.png)

`
2022-11-18T09:32:41+01:00 [ampache] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Application\StatisticGraph\ShowAction" for action "show"
2022-11-18T09:32:41+01:00 [****] (Ampache\Module\Application\StatisticGraph\ShowAction) -> Access denied, statistical graph disabled.
`

If I let that loaded page open (no reload!) and enable statistics again, two of the former empty pages for charts now contain a chart:
![grafik](https://user-images.githubusercontent.com/14225157/202657678-c4f9c33f-74e7-4ca6-8f68-942831d1993f.png)
So: type=catalog_size and type=catalog_files are now displayed and type=user_bandwidth and type=user_hits are still empty.

If I reload the Statistics page `/index.php#stats.php?action=show` it is back to square one (no chart, no maps, etc.)

**Comment by lachlan-00** [comment]
put more fixes in for this with f2cefb8ab

It's been a bit of a mess but it looks ok now.

**Comment by mitchray** [comment]
Seems to be resolved, reopen if not
